go-controller/pkg/libovsdbops/portgroup.go

```diff
@@ -161,12 +161,17 @@ func DeleteACLsFromPortGroupOps(nbClient libovsdbclient.Client, ops []libovsdb.O
 		return ops, nil
 	}
 
+	matchAcls, err := GetACLs(nbClient, acls)
```

Comment:
It is possible that someone uses `DeleteACLsFromPortGroupOps` with the ACLs UUIDs already available, and in that case it is unnecessary to run the expensive predicate. That is why I suggested that the caller of `DeleteACLsFromPortGroupOps` calls `GetACLs` first instead of this call being inside this method. This is how it is done elsewhere. Alternatively, you can call `GetACLs` for the subset of ACLs that don't have a UUID only.

Reply:
sorry that I missed your comments. I've made the change and pushed